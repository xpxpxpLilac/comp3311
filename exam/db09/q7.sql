drop view if exists q7;
create view q7
as
select * from 
(select count(*) from beers) as nbeers,
(select count(*) from brewers) as nbrewers,
(select count(*) from users) as nusers
;
