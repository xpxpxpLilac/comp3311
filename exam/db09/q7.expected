4959|1341|302
